scripts/train_torch_models_batch.sh

```
@@ -0,0 +1,294 @@
+    n_pickles="$(find "$folder" -maxdepth 1 -type f -name '*.pickle' | wc -l | tr -d ' ')"
+
+    if [[ "$n_pickles" == "0" ]]; then
+        echo "No .pickle files found for model '$model' in: $folder" >&2
+        echo "Expected either:" >&2
+        echo "  1) <folder>/*.pickle" >&2
+        echo "  2) <folder>/$model/*.pickle" >&2
+        exit 1
+    fi
+
+    local validation_out
+    if ! validation_out="$((uv run python - "$folder" <<'PY'
```

Shellcheck (0.11.0)

[error] Shells disambiguate $(( differently or not at all. For $(command substitution), add space after $( . For $((arithmetics)), fix parsing errors.

(SC1102)